test_runner/batch_others/test_wal_acceptor_async.py

```diff
@@ -0,0 +1,166 @@
+import asyncio
+import aiopg
+import random
+
+from fixtures.zenith_fixtures import WalAcceptor, WalAcceptorFactory, ZenithPageserver, PostgresFactory, Postgres
+from typing import List
+
+pytest_plugins = ("fixtures.zenith_fixtures")
+
+
+class BankClient(object):
+    def __init__(self, conn: aiopg.Connection, n_accounts=None, init_amount=None):
```

Comment:
According to the places which reference `n_accounts` and `init_amount`, `None` isn't going to work as a default value. Maybe there should be no defaults?

Reply:
I wanted to make args look like they are optional, but now I think that was not a great idea and it'll be better without defaults

Reply:
It looks like args can't be optional without default value, so I'll set default values to zero

Reply:
I still think those defaults aren't needed.

This is the only place where `.transfer` is called:

https://github.com/zenithdb/zenith/blob/d2912ecf9111113e052ea8babbc85c593b6c762f/test_runner/batch_others/test_wal_acceptor_async.py#L95

For it to be called, you create a `BankClient` acting as a mere wrapper of a connection object. Instead, you could make `transfer` a free function, since it doesn't really depend on properties of `BankClient`.